src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -1132,7 +1136,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         try {
             dataType = DataType.valueOf(command.getClass().getSimpleName());
         } catch (IllegalArgumentException e) {
-            logger.error("NODE {}: Command received with no implementation ({}).", nodeId,
+            logger.debug("NODE {}: Command received with no implementation ({}).", nodeId,
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.